### ISSUE TITLE
Improve the way to follow someone 

### ISSUE THREAD

**Issue by nicosomb** [elk-zone/elk#1020]
### Clear and concise description of the problem

When someone follows me, I think that finding the way to follow him/her is not so obvious. See the screenshot. 

![B3A87BCC-6AAB-4881-94FD-3762E92A0224](https://user-images.githubusercontent.com/121870/212155126-e8eab00c-9754-435a-957d-35805c060a85.jpeg)

The « follow area » tells me that the user follows me. 

### Suggested solution

Replace the text by « follow back ». 

Maybe it’s in fact a translation issue.

**Comment by cyberalien** [comment]
> Maybe it’s in fact a translation issue.

Original English text is `Follows you`, on hover changes to `Follow back`

**Comment by ZMYaro** [comment]
I know it would add another element to the screen, but should we consider adding a text label (like Twitter et al.) to say someone follows you.  I agree it is confusing when the button only shows its action on mouseover, especially when a lot of people access Elk from touch screens and cannot see the hover state.

**Comment by edimitchel** [comment]
I'm fully agreeing with @ZMYaro !
Also, as Mutuals doesn't work well for all languages, we should add that information that the account is following you.

**Comment by ayoayco** [comment]
Hover is also not very good for accessibility. We need the action to be clear in the button without needing to hover.

**Comment by edimitchel** [comment]
> Personally I'm in favour of just entirely replacing `Follows you` with `Follow back`

This could be made on french side

**Comment by boehs** [comment]
Redopvedin https://github.com/elk-zone/elk/commit/98a1b2dccb1de4052f775811ffa4d8bb3963c069

**Comment by boehs** [comment]
Want to see other languages in first

**Comment by edimitchel** [comment]
What do you mean? You want to get feedback from other languages?

ping @elk-zone/translators

**Comment by boehs** [comment]
> What do you mean? You want to get feedback from other languages?
> 
> ping @elk-zone/translators

French is a good start, just not enough for the issue to be closed I think

**Comment by prazdevs** [comment]
IMO, the button should always be "follow" or "follow back", and another mention out of the button could say "follows you". I think twitter has something like this ?

text change on hover is very bad ux/a11y.

**Comment by edimitchel** [comment]
I think getting another mention outside of the button would be IMHO a bed ux/a11y as the information is not clear.
That's said we could add the aria-label on the button and use "follows you" but the main text would be the action "follow back" which means indirectly the user is following you.

**Comment by juliemoynat** [comment]
Having an `aria-label` content different from the visible label is [a WCAG failure](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html). It's very bad for people using voice control.

I agree with @prazdevs idea. It's the better solution. The fact that someone is following you is just an information. It can't be a button name because there is no associated action with this name that people could understand.

**Comment by joelanman** [comment]
Just to say I agree that the current approach is not accessible due to relying on hover. For example if I use the keyboard to focus a button that says 'Following', there's no indication of what the button does. I only get that if I hover (Unfollow). I think the buttons should just say what they will do

**Comment by shuuji3** [comment]
I tried implementing changes in #2397 with the following features:
- Keep the same follow button label for better accessibility
- Add relationship badge: "Follows you"/"Mutuals" since there is no way to know if the account is following me after the above change.

Let me know if any feedback. 🙂

**Comment by joelanman** [comment]
@shuuji3 looks great thanks!
